### DISCUSSION TITLE
BluetoothLE Peripheral MAC Address

### DISCUSSION THREAD

**Discussion by wesoos** [shinyorg/shiny#886] [Q&A] [answered]
Hello,

How would I obtain the MAC address of the bluetooth device?  When I put a breakpoint in my scan results, I see a Native object of the peripheral which contains the address, but I can't access via code.

Please advise.

Thanks

**Comment by aritchie** [comment, marked as answer]
I don't expose the mac address at the moment as it is only available on Android.  I'm considering some options for a future release that may allow for this.

**Reply by wesoos** [reply]
Ok thank you..
